### PR TITLE
Use monochrome heart copy in push action to like a message

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Copy change as requested by design.

### CODE REVIEW

**Comment by daehn** [comment]
There will be another copy change for notifications when users like a message.

File: Resources/Base.lproj/Push.strings [hunk 1]
```diff
@@ -170,7 +170,7 @@
 "push.notification.action.message.reply" = "Reply";
 "push.notification.action.message.reply.button.title" = "Send";
 "push.notification.action.conversation.mute" = "Mute";
-"push.notification.action.message.like" = "❤️";
+"push.notification.action.message.like" = "♥";
```
Comment (typfel) [inline Resources/Base.lproj/Push.strings]: It's so tiny

**Comment by typfel** [comment]
Why the do not merge?

**Comment by daehn** [comment]
Will change it in the push when someone liked your message as well

**Comment by daehn** [comment]
The decision has been made, the heart in the push if someone else liked a message will stay the same.

**Comment by typfel** [comment]
What?!

**Pull request merged**